tests/libcxx/tests.broken

```diff
@@ -11,3 +11,7 @@
 ../../3rdparty/libcxx/libcxx/test/std/thread/futures/futures.async/async_race.38682.pass.cpp
 # The following test fails under clang-7 debug due to a memory leak.
 ../../3rdparty/libcxx/libcxx/test/libcxx/diagnostics/nodiscard_extensions.pass.cpp
+# The following three tests fails on ACC Windows VM.
```

Comment:
Do we know why these fail?

Reply:
I briefly mentioned in the descrption of the PR. They fail because of execution time differs between Linux and Windows environments. The tests currently use fixed time duration to determine pass or fail. I think we can create another issue for the problem.

Reply:
Sounds good. Can you create an issue (if there isn't one already) and link it in this thread in case someone comes across this later?

Reply:
Documented in #2409